### DISCUSSION TITLE
【求助】表格插入自定义元素，自定义元素的hoverbar不显示显示表格

### DISCUSSION THREAD

**Discussion by huangzhang-ops** [wangeditor-next/wangEditor-next#450] [Q&A] [answered]
## Bug Description

【求助】表格插入自定义元素，自定义元素的hoverbar不显示显示表格。这个有办法解决吗？
![image](https://github.com/user-attachments/assets/eea3f760-f764-4168-b2a8-58b69c591c94)

## What did you expect to happen?

*Please enter the details...*

## System, Browser, and Version

- Operating System
- Browser and Version

## wangEditor-next Version

*Please enter the details...*

## Can the demo reproduce the bug?

Yes/No

- Demo: https://cycleccc.github.io/demo/

## Online Demo

 - SandBox url: https://stackblitz.com/@cycleccc/collections/wangeditor

## Steps to Reproduce at Minimum Cost

- Step 1
- Step 2
- Step 3

**Comment by cycleccc** [comment, marked as answer]
你可以把对应的hoverbarkey 放到 toolbar 中，在 wangeditor 中 hoverbar 是单一的，选中了某个 node 就会展示这个 node 对应的 hoverbar key，在 table 中默认选中 table 展示 table 的 相关的 hoverbar key，可以拖拽选中自定义元素然后点击 toolbar 对应的 key 设置元素

**Comment by cycleccc** [comment]
hoverbar 为单一激活区域；表格内自定义元素可将对应 key 放入 toolbar 后操作。本讨论现关闭。
